internal/funcspec/spec.go

```diff
@@ -2,6 +2,7 @@ package funcspec
 
 import (
 	"context"
+	"github.com/hashicorp/waypoint-plugin-sdk/component"
```

Comment:
I think you're missing a some `go fmt` here.

Reply:
Done. `go fmt` doesn't handle the `stdlib \n external \n internal` for me - I've been watching out for it manually, but missed this one! Do you have a tool that arranges this for you?

Reply:
`vim-go` does this automatically for me.